src/themes/mainTheme.ts

```diff
@@ -17,11 +17,28 @@ enum Color {
   PrimaryLight = '#0D4DAD',
   TextPrimary = 'rgba(0, 0, 0, 0.87)',
   White = '#fff',
+  Registration = '#DAC48E',
 }
 
 const coreLocale = i18n.language === 'eng' ? coreEnUs : coreNbNo;
 const pickersLocale = i18n.language === 'eng' ? pickersEnUs : pickersNbNo;
 
+declare module '@mui/material/styles' {
+  interface Palette {
+    registration: PaletteColorOptions;
+  }
+
+  interface PaletteOptions {
+    registration: PaletteColorOptions;
+  }
+}
+
+declare module '@mui/material/Button' {
+  interface ButtonPropsColorOverrides {
+    registration: true;
+  }
+}
```

Comment:
Ønsker legge dette til `@types/mui.d.ts` slik vi har gjort det for andre tilfeller, men får det ikke til å virke for en eller annen grunn :/